lldb/source/Plugins/SymbolFile/DWARF/DWARFDebugInfo.h

```diff
@@ -58,6 +58,8 @@ class DWARFDebugInfo {
 
   const DWARFDebugAranges &GetCompileUnitAranges();
 
+  const std::shared_ptr<SymbolFileDWARFDwo> GetDwpSymbolFile();
```

Comment:
Remove const from by-value return. (it messes with move semantics and some other things) - or was this meant to return by reference? - yeah, I guess this latter, the underlying `m_dwarf.GetDwpSymbolFile()` seems to return by const reference, so this function probably should too?